src/vma/util/utils.cpp

```diff
@@ -1035,6 +1035,37 @@ int validate_ipoib_prop(const char* ifname, unsigned int ifflags,
 	}
 }
 
+//NOTE RAW_QP_PRIVLIGES_PARAM_FILE does not exist on ofed upstream
+int validate_raw_qp_privliges()
+{
+	// RAW_QP_PRIVLIGES_PARAM_FILE: "/sys/module/ib_uverbs/parameters/disable_raw_qp_enforcement"
+	char raw_qp_privliges_value = 0;
+	if (priv_read_file((const char*)RAW_QP_PRIVLIGES_PARAM_FILE, &raw_qp_privliges_value, 1, VLOG_DEBUG) <= 0) {
+		return -1;
+	}
+	if (raw_qp_privliges_value != '1') {
+		return 0;
+	}
+	return 1;
+}
+
+bool check_user_capabilities() 
+{
+	char resource_path[256];
+	sprintf(resource_path, PROC_STATUS_FILE, getpid());
+	char result[1024] = {0};
+	priv_safe_read_file(resource_path, result, 1024, VLOG_DEBUG);
+	char* cap_ptr = strstr(result, "CapEff:");
+	if (cap_ptr) {
```

Comment:
not clear from this code which capability are you searching for. 
preferably use defines
and definitely add comments